local.env.dist

```diff
@@ -23,17 +23,23 @@ LOGGING_LEVEL=
 # LOGGING_HANDLER default is stderr, or may be one of: syslog, file, errlog, stderr
 LOGGING_HANDLER=
 
+# Session store configuration. SESSION_STORE_TYPE can be "sql" or "memcache"
+SESSION_STORE_TYPE=
```

Comment:
Is this `SESSION_STORE_TYPE` env. var. one automatically used by SimpleSAMLphp, or do we need some custom code to respond to this?

Reply:
It's in the .../dockerbuild/ssp-overrides/config.php file.

```

    /*
     * Configure the data store for SimpleSAMLphp.
     *
     * - 'phpsession': Limited datastore, which uses the PHP session.
     * - 'memcache': Key-value datastore, based on memcache.
     * - 'sql': SQL datastore, using PDO.
     * - 'redis': Key-value datastore, based on redis.
     *
     * The default datastore is 'phpsession'.
     */
    'store.type'         => $SESSION_STORE_TYPE,
    ```